DOC-1007 Update to Initiate a SEPA Direct Debit transaction mutation

ntombing opened the pull request:
The initiateMerchantPaymentCollection mutation has been deprecated in favour of initiateSepaDirectDebitMerchantPayment. 

- Updated the mutation in API explorer

Guide:
- Updated steps 2 and 4 of the guide
- Updated the link in the guide
- Updated the mutation in the guide

racene-swan reviewed (changes requested):
Hi! A couple of changes: 
1. The Payload section content and the image don't match.
Here: Note the id (line 5) for your InternalSEPA Direct Debit payment.

Should it read: Note the paymentId? 

2. Please check that the inputs match the images, and the lines match too. mandateID looks like its on Line 4. And we first say paymentID is on line 4 - and then later that it's on line 5. 

3. Need an opening bracket before "line 5" in step 4. 

4. Generally just check that the lines and the inputs match the images. 

Thanks!

ntombing commented:
Hey!

- Changed `id` to `paymentID` in the payload. 
- Removed the comma after Choose `mandateId`

For the rest, I don't think anything needs to be changed. 

Payment `mandateId` and `paymentID` are different things, and I've referenced the correct lines for both.

> Need an opening bracket before "line 5" in step 4.
I don't think there was a change needed here.

Thank you!
